pkg/filemiddleware/middleware.go

```diff
@@ -10,23 +10,27 @@ import (
 )
 
 func NewResponder(next middleware.Responder, fname string, length int64) middleware.Responder {
-	return &fileMiddlewareResponder{
+	return &FileMiddlewareResponder{
 		next:     next,
 		fileName: fname,
 		length:   length,
 	}
 }
 
-type fileMiddlewareResponder struct {
+type FileMiddlewareResponder struct {
```

Comment:
I'll put it here just for future us, but it would be really nice to be able to validate responses in tests with `middleware.Responder` objects by just using the existing API (i.e. `WriteResponse`) rather than having the do type assertions every where in tests.